log/log.go

```diff
@@ -39,10 +41,63 @@ func (f levelFlag) Set(level string) error {
 	return nil
 }
 
+func setSyslogFormatter(appname, local string) error {
+	if appname != "" && local != "" {
+		return nil
+	}
+
+	fmter, err := newSyslogger(appname, local, origLogger.Formatter)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error creating syslog formatter: %v\n", err)
+		origLogger.Errorf("can't connect logger to syslogs: %v", err)
+		return err
+	}
+	origLogger.Formatter = fmter
+	return nil
+}
+
+func setJSONFormatter() {
+	origLogger.Formatter = &logrus.JSONFormatter{}
+}
+
+type logFormatFlag struct{}
+
+// String implements flag.Value.
+func (f logFormatFlag) String() string {
+	return origLogger.Level.String()
```

Comment:
so I'm just going to return the original log format uri, since thats probably the most useful